diff -c -r mackerel-agent-0.55.0.orig/Makefile mackerel-agent-0.55.0/Makefile
*** mackerel-agent-0.55.0.orig/Makefile	2018-06-20 11:17:18.000000000 +0900
--- mackerel-agent-0.55.0/Makefile	2018-06-23 22:27:09.000000000 +0900
***************
*** 6,11 ****
--- 6,12 ----
  BUILD_OS_TARGETS = "linux darwin freebsd windows netbsd"
  
  BUILD_LDFLAGS = "\
+ 	  -s -w \
  	  -X main.version=$(VERSION) \
  	  -X main.gitcommit=$(CURRENT_REVISION) \
  	  -X github.com/mackerelio/mackerel-agent/config.agentName=$(MACKEREL_AGENT_NAME) \
